### PR TITLE
zn/alternative hh vehicle sampling

### PR DESCRIPTION
**Pull request by zneedell**
start removing vehicles from households with extra vehicles first

---
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/1573)

### CODE REVIEW

**Comment by zneedell** [comment]
Test!

**Comment by wrashid** [comment]
Test!

**Comment by wrashid** [comment]
test!

File: src/main/scala/beam/utils/scenario/ScenarioLoader.scala [hunk 1]
```diff
@@ -104,78 +108,121 @@ class ScenarioLoader(
     var totalCarCount: Int = 0
 
     val scaleFactor = beamServices.beamConfig.beam.agentsim.agents.vehicles.fractionOfInitialVehicleFleet
```
Comment (wrashid) [inline src/main/scala/beam/utils/scenario/ScenarioLoader.scala]: move variable closer to where used (as it is not used in main content anymore)

**Pull request merged**